tests/e2e/tests/Map/AssertMapFunctionalitiesNotrigger.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { test } from '@playwright/test';
+import DashboardPage from 'Pages/DashboardPage';
+import MapComponent from 'Pages/MapComponent';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { NoTriggerDataSet } from 'testData/testData.enum';
+
+import { FloodsScenario } from '../../../../services/API-service/src/scripts/enum/mock-scenario.enum';
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    FloodsScenario.NoTrigger,
+    NoTriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(
+    NoTriggerDataSet.UserMail,
+    NoTriggerDataSet.UserPassword,
+  );
+});
+
+test(
+  qase(7, 'Verify Map functionality for no-triggered mode'),
```

Comment:
One thing I'm wondering about in general is how it is made clear what is actually tested in a test (and thereby what is not). In this case it is relatively clear by just looking at the code, but in some other cases it is sometimes less clear. For example in the aggregates-section above, there is much more you could potentially look at right? The numbers also changing on hover. Whether the numbers change to the right amount. Etc. Should we write this down in more detail, including maybe the things that are not tested yet? Do you have an opinion?

Reply:
I could improve on that. Finding clear titles on rather bigger tests is at times tricky.